### ISSUE TITLE
How can I disable the animation when ASCollectionView refreshing.

### ISSUE THREAD

**Issue by foolbear** [apptekstudios/ASCollectionView#86]
**Describe the bug**

I found an animation occurs when ASCollectionView refresh because of data updated or modal view disappear. How can I disable the animation.

There is no animation if just a static ASCollectionView(eg. RemindersScreen demo).

**To Reproduce**
Steps to reproduce the behaviour:

Here is modified source code of RemindersScreen, when tap the "Create new list", modal show the TagsScreen, when I dismiss TagsScreen, RemindersScreen jumps(animation) to refresh the content.

```struct RemindersScreen: View
{
	...
    
    @State private var sheetShowing = false                                     // <- modified

	var body: some View
	{
        NavigationView{
            ASCollectionView
            {   
....
                ASCollectionViewSection<Section>(id: .addNew)
                {
                    GroupSmall(model: self.addNewModel).onTapGesture {
                        self.sheetShowing = true                                     // <- modified
                    }
                }
...
            }
            .layout(self.layout)
            .contentInsets(.init(top: 20, left: 0, bottom: 20, right: 0))
            .alwaysBounceVertical()
            .background(Color(.systemGroupedBackground))
            .edgesIgnoringSafeArea(.all)
            .navigationBarTitle("Reminders", displayMode: .inline)
            .sheet(isPresented: $sheetShowing) {                                      // <- modified
                TagsScreen()
            }
        }
		
	}```

**Expected behaviour**
A description of what you expected to happen.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Xcode Version:**
- 

**Simulator, Device, Both?**
- Where is the problem occuring

**Comment by amrit42087** [comment]
I am facing same issue, Please let us know when ever any solution will be available

**Comment by cederache** [comment]
I made a PR allowing this : https://github.com/apptekstudios/ASCollectionView/pull/87

**Comment by foolbear** [comment]
@cederache thanks, I wait for next release.

**Comment by apptekstudios** [comment]
This fix is merged into master (thanks to @cederache). Waiting on a few other issues to be resolved and will push a new release, in the meantime master is fine to use. Note there has been a few environment modifiers renamed and some other renaming coming up so might need to update those for your project to build 👍
